app/controllers/provider_interface/content_controller.rb

```diff
@@ -53,5 +53,9 @@ def dates_and_deadlines
     def roadmap
       render_content_page :roadmap
     end
+
+    def organisation_permissions
+      render_content_page :organisation_permissions
```

Comment:
Does this page need breadcrumbs, similar to the dates and deadline one?
<img width="407" alt="image" src="https://user-images.githubusercontent.com/47917431/170437373-f665e963-6c4f-4f74-b104-2b34781198c9.png">

Reply:
:+1: Good catch yes probably does though not mentioned in trello.

Reply:
@JR-G https://github.com/DFE-Digital/apply-for-teacher-training/pull/7011/commits/91244c17a68a2bf80ba8c8e16fb0730d63d21a68